app/views/brexit_checker/_share_links.html.erb

```diff
@@ -1,7 +1,9 @@
 <section class="brexit-checker__share">
-  <h2 class="govuk-heading-m">
-    <%= I18n.t("brexit_checker.results.share_link_title") %>
-  </h2>
+  <%= render "govuk_publishing_components/components/heading", {
+    text: I18n.t("brexit_checker.results.share_link_title"),
```

Comment:
> you can just use t(... instead)

🤔  didn't know - will sort.

> There's no heading level on this call. I think the heading component default tag is h2 so that's not a big problem, however I think there's no harm in communicating that clearly to whomever is working on this in the future.

🤔   
I am inclined to agree as it helps at a glance, the gem docs / code sort-of suggest that everything that's _not_ a `h2` needs to be defined.   Do you think it's worth updating gem docs with this as well?

Reply:
I'm leaning towards not explicitly asking for h2 when that's the default, but have started a slack thread to garner comments and thoughts. It feels like it could snowball for components that have multiple optional params and reduces bloat to not add it here.

Reply:
If there are defaults I think we should use them - so leaving out `heading_level` parameter isn't a problem as it's [explained in the description of the component](https://components.publishing.service.gov.uk/component-guide/heading) and means that this is less verbose without compromising on its legibility.

Reply:
I can see both sides of this:
- At a glance you can tell the heading level without having gem knowledge that `h2` is the default
- Yet adding this adds extra code and if it comes with defaults this was intentional so best to leave.

Feels like the latter at the mo' so will leave _as is_ (unless slack chat results in a stronger feeling the other way)

Reply:
I'm willing to drop this. Re-reading [the docs](https://components.publishing.service.gov.uk/component-guide/heading) is communicates that it's a h2 by default so less verbosity is good.